.github/workflows/renew_certificate.yaml

```diff
@@ -0,0 +1,48 @@
+name: Renew the CA-verified Certificate
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Runs at 00:00 on day-of-month 1 in every 2nd month. (see https://crontab.guru)
+    - cron: "0 0 1 */2 *"
+
+jobs:
+  build:
+    name: Trigger Juju renew certificate action on certbot-k8s
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Juju
+        run: |
+          sudo snap install juju --classic
+
+      # We need the kubeconfig and the controllers.yaml file into our environment
+      # in order to login and refresh the charms.
+      - name: Adding Credentials
+        shell: bash
+        run: |
+          mkdir -p ~/.kube
+          mkdir -p ~/.local/share/juju
+          echo "${KUBECONFIG_B64}" | base64 -d > ~/.kube/config
+          echo "${CONTROLLERS_B64}" | base64 -d > ~/.local/share/juju/controllers.yaml
+
+          echo "${CONTROLLER_PASSWORD}" | juju login -c finos-legend -u admin
+
+        env:
+          KUBECONFIG_B64: "${{ secrets.KUBECONFIG_B64 }}"
+          CONTROLLERS_B64: "${{ secrets.CONTROLLERS_B64 }}"
+          CONTROLLER_PASSWORD: "${{ secrets.CONTROLLER_PASSWORD }}"
+
+      - name: Renewing Certificate
+        run: |
+
+          # Controller and model names are the same, "finos-legend"
+          juju switch finos-legend:finos-legend
+                    
+          # Running juju status will show us the current revisions of the charms.
+          juju status --relations
+
+          # Run the renew action on the certbot-k8s charm.
+          juju run-action certbot-k8s/0 renew-certificate --wait
```

Comment:
So, ideally, we would check for the result of the action. According to the documentation (https://juju.is/docs/olm/working-with-actions), there are 5 possible values for the status of an action.

The idea is to fail the github action if the juju action did not complete successfully, which will then show us if something went wrong or not on a simple glance. Currently, this github action would be green even if the juju action failed, which isn't intuitive.

tl;dr: we should check the juju action result. If it's not successful, we should fail the github action / exit 1.